projects/packages/videopress/src/class-status.php

```diff
@@ -38,6 +38,6 @@ public static function is_jetpack_active() {
 	 * @return boolean
 	 */
 	public static function is_standalone_plugin_active() {
-		return class_exists( 'Jetpack_Videopress_Plugin' );
+		return class_exists( 'Jetpack_VideoPress_Plugin' );
```

Comment:
need to rename the class declaration as well and check all occurrences calling it